samples/csharp/MicrosoftGraphSDK/Submission.cs

```diff
@@ -157,5 +188,115 @@ public class Submission
                 throw new GraphException($"ReassignAsync call: {ex.Message}", ex, classId, assignmentId, submissionId);
             }
         }
+
+        /// <summary>
+        /// Creating Feedback resource outcome
+        /// </summary>
+        /// <param name="client">Microsoft Graph service client</param>
+        /// <param name="classId">User class id</param>
+        /// <param name="assignmentId">Assignment id in the class</param>
+        /// <param name="submissionId">Student submission id</param>
+        /// <returns>EducationSubmission</returns>
+        public static async Task<EducationOutcome> CreateFeedbackResourceOutcome(
+            GraphServiceClient client,
+            string classId,
+            string assignmentId,
+            string submissionId)
+        {
+            var educationOutcome = new EducationFeedbackResourceOutcome
+            {
+                OdataType = "#microsoft.graph.educationFeedbackResourceOutcome",
+                FeedbackResource = new EducationWordResource
+                {
+                    OdataType = "#microsoft.graph.educationWordResource",
+                    DisplayName = "Document1.docx"
+                }
+            };
+
+            return await client.Education
+                .Classes[classId]
+                .Assignments[assignmentId]
+                .Submissions[submissionId]
+                .Outcomes
+                .PostAsync(educationOutcome);
```

Comment:
wrap this code with a try,catch block as you did for the **PatchOutcomeAsync**

Reply:
wrapped the code in try catch block